goal_src/jak2/build/all_objs.json

```diff
@@ -0,0 +1,2252 @@
+["precursor-stone-ag-LOUTCSTB",                "precursor-stone",            4,["LOUTCSTB"], "levels/outro"],
+["precursor-stone-ag-NEB",                     "precursor-stone",            4,["NEB"], "levels/metal_head_nest/boss_room"],
+["precursor-stone-ag-COA",                     "precursor-stone",            4,["COA"], "levels/construction_site"],
+["loutcstb",                                   "loutcstb",                   4,["LOUTCSTB"], "levels/outro"],
+["tpage-2953",                                 "tpage-2953",                 4,["LASHTHRN"], "levels/palace/throne_room"],
+["tpage-3266",                                 "tpage-3266",                 4,["LASHTHRN"], "levels/palace/throne_room"],
+["lashthrn",                                   "lashthrn",                   4,["LASHTHRN"], "levels/palace/throne_room"],
+["tpage-1598",                                 "tpage-1598",                 4,["TOC"], "levels/mars_tomb/right"],
+["tpage-1599",                                 "tpage-1599",                 4,["TOC"], "levels/demo"],
+["tpage-1832",                                 "tpage-1832",                 4,["TOC"], "levels/mars_tomb/right"],
+["water-anim-tomb-ag",                         "water-anim-tomb",            4,["TOC", "TOA"], "levels/mars_tomb"],
+["tomb-baby-spider-ag",                        "tomb-baby-spider",           4,["TOC", "TOE", "TOB"], "levels/mars_tomb"],
+["tomb-beetle-ag",                             "tomb-beetle",                4,["TOC"], "levels/mars_tomb/right"],
+["tomb-door-ag",                               "tomb-door",                  4,["TOC"], "levels/mars_tomb/right"],
+["tomb-vibe-ag",                               "tomb-vibe",                  4,["TOC"], "levels/mars_tomb/right"],
+["tomb-simon-button-ag-TOC",                   "tomb-simon-button",          4,["TOC"], "levels/mars_tomb/right"],
+["tomb-simon-button-ag-TOA",                   "tomb-simon-button",          4,["TOA"], "levels/mars_tomb"],
+["tomb-plat-simon-ag",                         "tomb-plat-simon",            4,["TOC"], "levels/mars_tomb/right"],
+["tomb-wing-door-ag-TOC",                      "tomb-wing-door",             4,["TOC"], "levels/mars_tomb/right"],
+["tomb-wing-door-ag-TBO",                      "tomb-wing-door",             4,["TBO"], "levels/mars_tomb/baron"],
+["tomb-wing-door-ag-TOA",                      "tomb-wing-door",             4,["TOA"], "levels/mars_tomb"],
+["tomb-button-ag-TOC",                         "tomb-button",                4,["TOC"], "levels/mars_tomb/right"],
+["tomb-button-ag-TOB",                         "tomb-button",                4,["TOB"], "levels/mars_tomb/left"],
+["tomb-beetle-door-ag",                        "tomb-beetle-door",           4,["TOC"], "levels/mars_tomb/right"],
+["tombc-vis",                                  "tombc-vis",                  4,["TOC"], "levels/mars_tomb/right"],
```

Comment:
why not prefer `right` or `left` instead of the cryptic `tombc` and `tombb`?

Reply:
I guess I don't personally find it that cryptic... maybe it is best to use `left` and `right`. It looks like `tombd` doesn't have any code files so that one is irrelevant, and `tombe` has... one.

Reply:
I lied, `tombe` has 2 actually.